pinot-common/src/main/java/org/apache/pinot/common/utils/ServiceStartableUtils.java

```diff
@@ -61,6 +65,7 @@ public static void applyClusterConfig(PinotConfiguration instanceConfig, String
         .build();
     zkClient.waitUntilConnected(zkClientConnectionTimeoutMs, TimeUnit.MILLISECONDS);
 
+    setupTimezone(instanceConfig);
```

Comment:
Oh one of the reason I put it there is because the method might return prematurely in line 74 if there is nothing found under `CLUSTER_CONFIG_ZK_PATH_TEMPLATE` path. Hence, timezone is not set. Wdyt ?

Reply:
right, my feeling here is that if users doesn't set up time zone from cluster config ,then it's pretty hard to

Reply:
Hmm so what do you suggest we do in this case? Should we call `setupTimezone(instanceConfig)` twice: in both line 73 before the `return` and in line 94 at the end of the method